purchase_default_terms_conditions/models/purchase.py

```diff
@@ -10,7 +11,9 @@ class PurchaseOrder(models.Model):
 
     @api.onchange("partner_id", "company_id")
     def onchange_partner_id(self):
-        if self.partner_id.purchase_note:
+        if self.partner_id.purchase_note and not is_html_empty(
```

Comment:
You can remove `self.partner_id.purchase_note` because `is_html_empty` can check for the case the case field is empty or not as well. It would be nice if you can try and check it again.
thank you

Reply:
Done @quannguyen156. Can you check please?